### DISCUSSION TITLE
Monterey 12.3.1 fresh install, brew python3.9 doesnt work

### DISCUSSION THREAD

**Discussion by nolrinale** [Homebrew/discussions#3212] [Getting started]
Hello, today I deleted my whole M1 macbook air and started a new fresh Monterey installation with 12.3.1, installed brew and proceed to install python with `brew install python` , the installation ended successfully installing Python 3.9.12

However after typing `python3 --version` the output I get is `Python 3.8.9` the default Monterey one.
A similar situation also happens with pip3 as I'm expecting to use the brew one, instead, I get after typing `pip3 --version` I the output `pip 20.2.3 from /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages/pip (python 3.8)`

How can I make homebrew once again take control python and pip and be able to access the apps by just typing the name like before?

**Comment by gromgit** [comment]
That's bizarre, that you could run `brew` but not Homebrew's `python3`, since both would've been found in the same directory.

EDIT: After my morning coffee, of course it's not _that_ bizarre. It's easily explained by the order of directories in your `PATH`. :grinning: 

What's the output of:
```
which -a brew
which -a python3
which -a pip3
brew config
brew doctor
```
?

**Reply by nolrinale** [reply]
Well you gonna be surprised, but I was suddenly prompted to install the latest command line tools twice (versions 13.2 and 13.3 i think...) and after restarting the system suddenly it began to work!!

The current outputs are now these: (but before the restart i was getting only the default ones `/usr/bin/python3` and `/usr/bin/pip3`

```
which -a brew
/opt/homebrew/bin/brew
```

```
which -a python3
/opt/homebrew/bin/python3
/usr/bin/python3
```

```
which -a pip3
/opt/homebrew/bin/pip3
/usr/bin/pip3
```

I could try `brew uninstall python` but honestly since it began to magically work I dont dare to do it again ....
